lms/djangoapps/discussion_api/forms.py

```diff
@@ -51,6 +52,14 @@ class ThreadListGetForm(_PaginationForm):
     topic_id = TopicIdField(required=False)
     text_search = CharField(required=False)
     following = NullBooleanField(required=False)
+    order_by = ChoiceField(
+        choices=[(choice, choice) for choice in ["last_activity_at", "comment_count", "vote_count"]],
```

Comment:
Nit: it would be nice if the possible values were just defined in one place (as opposed to both here and in api.py). Not sure if there is a way for api.py to reference these values.